newjersey/navigator.business.nj.gov
# feat: [#187399321] replace blockquotes with conditional callouts

## Description



Update all Markdown and JSON files that use block quote syntax to use the new conditional callout syntax. This transforms the appearance from the "green box" style to the conditional callout boxes, with icons.

Additionally, update the component syntax for the Callout component to better handle default states, and update the `[filingUrlSlug].tsx` and `CannabisPriorityRequirements.tsx` files which previously directly used green boxes directly.

### Ticket



[#187399321](https://www.pivotaltracker.com/story/show/187399321)

### Approach



Wrote a Python script to make the changes; this is now located in the `content/scripts` directory. Tests for this file to come later.

### Steps to Test



Navigate to any of these changed files in the dev site and on local and compare the two to see their style change. Additionally, for the files that used the green boxes directly, you can see the changes here:

http://localhost:3000/filings/1094
http://localhost:3000/tasks/priority-status-cannabis (select a category that would give priority such as disabled veteran, then click "View Application Needs")

### Notes



We may need to keep track of all the files changed, their location in the CMS, and their location in the app, if the content team wants to review each of these.

Before we merge this in, we should merge in the latest from `content-repo` to prevent merge conflicts.

## Code author checklist
